doc/Getting-started/index.md

```diff
@@ -65,543 +41,211 @@ To use Kaa Sandbox, your system must meet the following minimum system requireme
+      | Protocol | Port | RangeSource|
+      |----------|------|------------|
+      | TCP      | 22   | 0.0.0.0/0  |
+      | TCP      | 8080 | 0.0.0.0/0  |
+      | TCP      | 9999 | 0.0.0.0/0  |
+      | TCP      | 9998 | 0.0.0.0/0  |
+      | TCP      | 9997 | 0.0.0.0/0  |
+      | TCP      | 9889 | 0.0.0.0/0  |
+      | TCP      | 9888 | 0.0.0.0/0  |
+      | TCP      | 9887 | 0.0.0.0/0  |
+      | TCP      | 9080 | 0.0.0.0/0  |
 
-   ```json
-   {
-     "type": "record",
-     "name": "Notification",
-     "namespace": "org.kaaproject.kaa.schema.example",
-     "fields":[
-        {
-          "name": "message",
-          "type": "string"
-        }
-      ]
-    }
-```
+   * Select the created security group.
 
-2. Open the admin UI in your browser ( [http://127.0.0.1:8080](http://127.0.0.1:8080/) ) and log in as a tenant developer (user/password: devuser/devuser123).
-3. Open the relevant **Notification schemas** window (**Applications =\> My First Kaa Application =\> Schemas =\> Notification**) and click **Add new schema**.
-4. In the **Add new schema** window, fill in the fields as shown in the following screenshot and then click **Add**.   
-<img src="attach/image2015-6-15 20-9-14.png" width="800" height="400">
+7. On **Review Instance Launch** step, make any changes as appropriate.
 
-5. Scroll down and use the Upload from file function to find the previously created json file with the schema. Alternatively, you can use the Schema [Avro UI form***]() to create the schema.
-6. Click Upload.  
-<img src="attach/image2015-6-19 14-51-26.png" width="900" height="500">
-7. Click Add at the top of the window.
+8. Click **Launch** and then in the **Select an existing key pair** or **Create a new key pair** dialog, do one of the following:
 
-As a result of this operation you will see two notification schemas in the list:
-<img src="attach/image2015-6-15 20-18-54.png" width="1200" height="300">
+   * Select an existing key pair from the **Select a key pair** drop list.
 
-In this screenshot, version 2.0 is the notification schema that was just created. We will use this version for the SDK generation in the next step.
+   * If you need to create a new key pair, click **Create a new key pair**. Then create the new key pair as described in [Creating a key pair](http://docs.aws.amazon.com/gettingstarted/latest/wah/getting-started-prereq.html).
 
-## Generate SDK
+9. Click **Launch Instances**. The **Launch Status** page will be displayed.
 
-To generate the SDK for the new application, proceed as follows:
+10. Click **View Instances**.
 
-1. Select the **My First Kaa Application** application and click **Generate SDK**.   
-<img src="attach/sdk_gen0.png" width="494" height="430">
+11. After launching Kaa Sandbox instance, go to **<your\_instance\_public\_dns\>:9080/sandbox** or **<your\_instance\_public\_ip\>:9080/sandbox** URL. Public DNS or IP of your instance are available from your instance description.
```

Comment:
What does `your_instance_public_dns` mean? I guess you have in mind *domain name*, not *domain name server*.